src/Adapters/Storage/Mongo/MongoStorageAdapter.js

```diff
@@ -159,27 +159,38 @@ export class MongoStorageAdapter {
     .then(collection => collection.insertOne(mongoObject));
   }
 
-  // Remove the object with the given ID. If the object does not exist, reject with OBJECT_NOT_FOUND.
-  // If there is a different type of error, reject with INTERNAL_SERVER_ERROR and a message
-  // that doesn't leak info to the client.
-  deleteObject(className, objectId) {
+  // Remove all objects that match the given parse query. Parse Query should be in Parse Format.
+  // If no objects match, reject with OBJECT_NOT_FOUND. If objects are found and deleted, resolve with nothing.
+  // If there is some other error, reject with INTERNAL_SERVER_ERROR.
+
+  // Currently accepts the acl, schemaController, validate
+  // for lecacy reasons, Parse Server should later integrate acl into the query. Database adapters
+  // shouldn't know about acl.
+
+  // Currently resolves with empty object because thats what HooksController expects, but that should change.
```

Comment:
If you want, sure. I think the best behaviour is probably to resolve with undefined.